src/components/Hero.tsx

```
@@ -0,0 +1,43 @@
+import { Button } from "@/components/ui/button";
+import { Mail } from "lucide-react";
+import Link from "next/link";
+import { GithubIcon } from "./icons/GithubIcon";
+import { LinkedinIcon } from "./icons/LinkedinIcon";
+
+export function Hero() {
+  return (
+    <section className="flex flex-col items-center justify-center min-h-[60vh] text-center">
+      <div className="space-y-4">
+        <h1 className="text-4xl md:text-6xl font-bold tracking-tight">
+          Hi, I'm Jonathan Rodiger
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)